lms/static/js/student_account/views/account_settings_factory.js

```diff
@@ -60,12 +65,14 @@
                             view: new AccountSettingsFieldViews.PasswordFieldView({
                                 model: userAccountModel,
                                 title: gettext('Password'),
-                                screenReaderTitle: gettext('Reset your Password'),
+                                screenReaderTitle: gettext('Reset Your Password'),
                                 valueAttribute: 'password',
                                 emailAttribute: 'email',
-                                linkTitle: gettext('Reset Password'),
+                                linkTitle: gettext('Reset Your Password'),
                                 linkHref: fieldsData.password.url,
-                                helpMessage: gettext('When you click "Reset Password", a message will be sent to your email address. Click the link in the message to reset your password.')
+                                helpMessage: gettext('When you click "Reset Your Password", edX will send a message ' +
```

Comment:
@waheedahmed Note that it is important that all UI changes have their i18n support tested on a sandbox as that would catch problems like this. See step 7 of the general guidelines here:

https://openedx.atlassian.net/wiki/display/TNL/Code+Review+Guidelines+and+Gotchas